docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
     volumes:
       - ./:/fola
       - /fola/node_modules
-    command: npx nodemon -w /fola/app -w /fola/middleware -L server.js
+    command: npx nodemon -w /fola/server.js -w /fola/app -w /fola/middleware -L server.js
```

Comment:
adding server.js to the watch list so nodemon restarts on changes to that file too